### PR TITLE
added codebyte to unshift and valueOf method for arrays in javascript

### PR DESCRIPTION
**Pull request by yashladekar**
### Description

#2278
This PR adds code bytes for unshift and valueOf  methods

This code examples demonstrate the use of the unshift and valueOf methods in JavaScript . The unshift example shows how to add elements to the beginning of an array, while the valueOf example shows how to 
return the array itself.

### Type of Change

- Editing an existing entry (fixing a typo, bug, issues, etc)
- Updating the documentation

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/javascript/concepts/arrays/terms/unshift/unshift.md [hunk 1]
```diff
@@ -50,3 +50,14 @@ countToTen.unshift(1, 2);
 console.log(countToTen);
 // Output: [1, 2, 3, 4, 5, 6, 7, 8, 9, 10]
 ```
+
+### Codebyte Example
+
+The code below adds `Lemon` and `Pineapple` to the beginning of the array `fruits` with the help of the unshift() method.
```
Comment (SSwiniarski) [inline content/javascript/concepts/arrays/terms/unshift/unshift.md:56]:
```suggestion
The code below adds `Lemon` and `Pineapple` to the beginning of the array `fruits` with the help of the `unshift()` method.
```

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@yashladekar I did a small edit and had one comment for you.

**Review by SSwiniarski — APPROVED** [review]
@yashladekar Looks good for second review.

File: content/javascript/concepts/arrays/terms/unshift/unshift.md [hunk 2]
```diff
@@ -50,3 +50,14 @@ countToTen.unshift(1, 2);
 console.log(countToTen);
 // Output: [1, 2, 3, 4, 5, 6, 7, 8, 9, 10]
 ```
+
+### Codebyte Example
+
+The code below adds `Lemon` and `Pineapple` to the beginning of the array `fruits` with the help of the `unshift()` method.
```
Comment (caupolicandiaz) [inline content/javascript/concepts/arrays/terms/unshift/unshift.md:56]:
```suggestion
The code below adds `Lemon` and `Pineapple` to the beginning of the array `fruits` with the help of the `.unshift()` method.
```

File: content/javascript/concepts/arrays/terms/unshift/unshift.md [hunk 3]
```diff
@@ -50,3 +50,14 @@ countToTen.unshift(1, 2);
...
+
+```codebyte/js
+
```
Comment (caupolicandiaz) [inline content/javascript/concepts/arrays/terms/unshift/unshift.md:59]: remove the empty line at the top of the code block

File: content/javascript/concepts/arrays/terms/unshift/unshift.md [hunk 4]
```diff
@@ -50,3 +50,14 @@ countToTen.unshift(1, 2);
 console.log(countToTen);
 // Output: [1, 2, 3, 4, 5, 6, 7, 8, 9, 10]
 ```
+
+### Codebyte Example
```
Comment (caupolicandiaz) [inline content/javascript/concepts/arrays/terms/unshift/unshift.md:54]:
```suggestion
## Codebyte Example
```

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@yashladekar I've completed my initial review. Please update the entry, and tag me when you've finished making changes. Thanks

**Review by yashladekar — COMMENTED** [review]
The changes are done. Do let me know if there are any other changes I need to make.

**Pull request merged**